spec/controllers/alchemy/admin/pages_controller_spec.rb

```diff
@@ -718,6 +718,7 @@ module Alchemy
 
           before do
             allow(request).to receive(:port).and_return(3000)
+            allow(@controller).to receive(:request).and_return(request)
```

Comment:
This can't be real. This must be handled by `rspec-rails`. Controller specs are truly broken in Rails 5. The only way to fix this is to rewrite these tests as request specs.